bin/gwdetchar-overflow

```diff
@@ -200,10 +201,14 @@ for dcuid in args.dcuid:
                               desc='        Going deep'):
             data = get_data(channels, s, e, **read_kw)
             for ch in channels:
-                overflows[ch] += daq.find_overflow_segments(
-                    data[ch],
-                    cumulative=True,
-                )
+                try:
+                    overflows[ch] += daq.find_overflow_segments(
+                        data[ch],
+                        cumulative=True,
+                    )
+                except Exception as exc:
```

Comment:
Please identify the expected error modes here, rather than just `Exception`, since we only want to catch things associated with channels missing from the source data.